lib/v2/statsgov/zxfb.js

```diff
@@ -0,0 +1,43 @@
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'http://www.stats.gov.cn';
+    const url = `${rootUrl}/tjsj/zxfb/`;
```

Comment:
Please make it more generic so devs do not need to reinvent the wheels when somebody wants RSS for the same site like `http://www.stats.gov.cn/tjgz/tjdt/`